opencodelists/forms.py

```diff
@@ -54,7 +55,17 @@ def form_field_from_model(model_class, field_name, **kwargs):
       appropriate for their specific form."""
     model_field = model_class._meta.get_field(field_name)
     field_class = model_field.formfield
-    return field_class(validators=model_field.validators, **kwargs)
+    validators = model_field.validators
+    for validator in validators:
+        if isinstance(validator, RegexValidator):
+            kwargs["widget"] = forms.TextInput(
+                attrs={
+                    "pattern": f".*{validator.regex.pattern}.*",
+                    "title": validator.message,
+                }
+            )
```

Comment:
thought: We could maybe validate that we only have one RegexValidator here and raise an exception otherwise, but it's probably fine as is.